@navikt/ds-css/grid/grid.css

```diff
@@ -0,0 +1,17 @@
+.navds-grid {
+  display: grid;
+  grid-template-columns: repeat(12, minmax(0, 1fr));
+  grid-gap: var(--navds-grid-gutter-small);
+}
+
+@media (min-width: 448px) {
+  .navds-grid {
+    grid-gap: var(--navds-grid-gutter-medium-and-larger);
+  }
+}
+
+@media (min-width: 648px) {
+  .navds-grid {
+    grid-gap: var(--navds-grid-gutter-medium-and-larger);
```

Comment:
Både `min-width: 448px` og `min-width: 648px` bruker `grid-gap: var(--navds-grid-gutter-medium-and-larger)`. Er det en typo?

Reply:
Tanken var at det gjelder fra medium og oppover, men det er kanskje forvirrende. Forslag til navneendring?

Reply:
Åja, sånn sett. Heil greit navn, men jeg synes det er forvirrende å repetere `grid-gap: var(--navds-grid-gutter-medium-and-larger)`, for det holder jo å definere den én gang i `min-width: 448px`.

Reply:
Ah, ja, det la jeg ikke merke til. Fikser :)